app/templates/components/billing/payment.hbs

```diff
@@ -115,6 +115,9 @@
             <p>{{stripeError.message}}</p>
           {{/if}}
         </StripeCard>
+        <span class="trial-no-charge-text">
+          Your credit card will not be charged for 14 days. Enjoy your trial!
+        </span>
```

Comment:
This message should be shown for users starting a trial, but I don't see any condition for that.

Reply:
@DominikAlberski thanks, I added it.

Reply:
@DominikAlberski thanks, added.